tensorflow/tools/compatibility/ast_edits.py

```diff
@@ -29,6 +29,8 @@
 import pasta
 import six
 
+import IPython.core.inputtransformer2 as ip
```

Comment:
I'd rather avoid a hard dependency. Can you put this import into a try block and make the processing based on it conditional on it succeeding?

Reply:
I just adde change to my master I guess it needs some time to update on pull request